### PR TITLE
luci-app-vpnbypass: luci tree Makefile + i18n support.

### PR DESCRIPTION
**Pull request by stangri**
Sorry, forgot to update Makefile when I moved luci-app-vpnbypass into luci tree + initial i18n support.

Signed-off-by: Stan Grishin <stangri@melmac.net>

### CODE REVIEW

File: applications/luci-app-vpnbypass/po/templates/vpnbypass.pot [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+msgstr ""
+
+msgid "Local IP Subnets to Bypass"
+msgstr ""
+
+msgid "Local IP ranges with direct internet access (outside of the VPN tunnel)"
+msgstr ""
+
+msgid "Reverse Local IP Subnets"
+msgstr "Local IP ranges with internet access via VPN (everything else will go outside of the VPN tunnel)"
```
Comment (hnyman) [inline applications/luci-app-vpnbypass/po/templates/vpnbypass.pot]: These lines 31-32 look strange.  Looks like a copy-paste error.

Additionally the strings should be in alphabetical order. (the next i18n.sync run will organise them, but you might do that by yourself. Just run ./build/i18n-sync.pl in your Luci root dir, and see what happens to the file. The include that sorted file in the commit.

File: applications/luci-app-vpnbypass/Makefile [hunk 2]
```diff
@@ -3,42 +3,11 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=luci-app-vpnbypass
-PKG_VERSION:=1.0.0
-PKG_RELEASE:=3
-PKG_LICENSE:=GPL-3.0+
-PKG_MAINTAINER:=Stan Grishin <stangri@melmac.net>
```
Comment (hnyman) [inline applications/luci-app-vpnbypass/Makefile]: You might leave the maintainer line here. Some newer LuCI apps have the maintainer listed, especially in this kind of case when you are really doing both the main app and the LuCI app.

**Comment by stangri** [comment]
Hannu,
I'd also prefer to leave the PKG_VERSION/PKG_RELEASE in the makefile to be in sync with the main app -- would that be acceptable?
Thank you for your comment -- I'll rebuild the pot-file.

**Comment by hnyman** [comment]
> I'd also prefer to leave the PKG_VERSION/PKG_RELEASE in the makefile to be in sync with the main app -- would that be acceptable?

Sure. Most apps do not carry that info and just let the automatic Luci versioning do its magic, but some do provide a native version. See e.g. ddns and aria2 apps.

**Comment by stangri** [comment]
Thank you Hannu, I've rebuilt the pot-file and edited Makefile.

**Comment by hnyman** [comment]
One hint for next time: when you rebase / squash commits, edit also the combined commit message... No need to have several signed-off-by lines in the message  ;-)

**Pull request merged**